package.json

```diff
@@ -1,15 +1,33 @@
 {
     "dependencies": {
-        "@project-serum/anchor": "^0.20.1"
+        "@project-serum/anchor": "^0.20.1",
+        "@solana/wallet-adapter-base": "^0.9.3",
+        "@solana/wallet-adapter-react": "^0.15.3",
+        "@solana/wallet-adapter-react-ui": "^0.9.5",
+        "@solana/wallet-adapter-wallets": "^0.15.4",
+        "@solana/web3.js": "^1.34.0",
+        "daisyui": "^2.2.2",
+        "next": "12.1.0",
+        "react": "17.0.2",
+        "react-dom": "17.0.2"
```

Comment:
None of these should be needed other than `anchor`, because the root level is only handling our contract code. Only our `app/` cares about the FE specific dependencies